.github/workflows/main.yml

```diff
@@ -8,24 +8,23 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
       matrix:
         image:
           - "centos:8"
           - "fedora:34"
```

Comment:
why don't we test it on other versions of each OS? 

if we only use the latest one, shouldn't fedora be 36?

Reply:
We perform smoke test only against single version of OS to make sure the installer works, it will work for other versions. Testing all versions is a job of delivery-checker.

I'm not sure Fedora 36 works, so I make test to check the functionality is still correct, not to check something new.

Reply:
Fedora 36 packages are not published yet. Nothing to test.

Reply:
The same for Ubuntu Jammy